spinalcordtoolbox/centerline/core.py

```diff
@@ -237,10 +238,30 @@ def get_centerline(im_seg, param=ParamCenterline(), verbose=1, remove_temp_files
         fname_ctr = (add_suffix(os.path.basename(im_seg.absolutepath), "_ctr") if im_seg.absolutepath
                      else "centerline.nii.gz")
         im_centerline.save(os.path.join(tmp_folder, fname_ctr), mutable=True)
-
+    
+    # Obtain centerline coordinates (`Centerline.points`) and derivatives (`Centerline.derivatives`) in voxel ("pix") space and RPI orientation
+    arr_ctl=np.array([x_centerline_fit, y_centerline_fit, z_ref])
+    arr_ctl_der=np.array([x_centerline_deriv, y_centerline_deriv, np.ones_like(z_ref)])
+    
+    # If specified by user, adjust centerline coordinates (`Centerline.points`) and derivatives (`Centerline.derivatives`) to physical ("phys") space and native (`im_seg`) orientation
+    if  space!='pix':
```

Comment:
```suggestion
    if  space != 'pix':
```
PEP8 nitpick https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements